### ISSUE TITLE
Would anyone be interested in a dist or min file in the repo for working with CodePen and Unpkg.com?

### ISSUE THREAD

**Issue by lasergoat** [airbnb/react-dates#426]
If enough people are interested, I can PR a change which will add a compiled version into the repo for sites like unpkg.com.

The main benefit of this would be to allow issues opened on this repo to require a codepen replication of the issue. 

In code pen, you can add dependancies using unpkg.com but since react-dates doesn't have a compiled version, you cannot. React-dates only has node code which must go through webpack before it will work in a project.

An example of this is here: http://codepen.io/davidkpiano/pen/yJwmEa
We can also add an issue template like in https://github.com/davidkpiano/react-redux-form/blob/master/.github/ISSUE_TEMPLATE.md

**Comment by ljharb** [comment]
Having a codepen or jsfiddle is great - does wzrd.in help here?

I would strongly prefer not including a compiled bundle in either the repo or the npm package.

**Comment by lasergoat** [comment]
I don't believe it's actually in the repo, but we use a config on travis or something to build it as a umd. It's something I'm new too. @davidkpiano how did you do it for react-redux-form

**Comment by ljharb** [comment]
Specifically i meant https://wzrd.in/standalone/react-dates@latest

**Comment by lasergoat** [comment]
@ljharb oh ok, I see.

here's a decent start:

http://codepen.io/polyesterhat/pen/gmJdrY?editors=0010

I'm not sure what's wrong with it currently

**Comment by ljharb** [comment]
@lasergoat I edited it a bit - http://codepen.io/airbnb/pen/LWoXQe?editors=0010 the CSS still needs updating, but hopefully that will help unblock you.

**Comment by lasergoat** [comment]
any ideas on how to get the css in there?

**Comment by MDB-20** [comment]
@lasergoat   this should fix import 'react-dates/lib/css/_datepicker.css';

**Comment by Hellenic** [comment]
Thanks guys for the template!
Used it and added in the CSS as well. If you need, feel free to fork: https://codepen.io/hellenic/pen/KvwGPb

(Note: Input behaves a bit odd for me, and actually need to click at the outer border of the input to open the DatePicker)

**Comment by jamesstoneco** [comment]
I see this issue is quite old but is still marked as open. I couldn't find any codepens (including those listed above) that are currently working. I was wanting to create some quick isolated prototypes, so I created the codesandbox below.

https://codesandbox.io/s/react-dates-demo-wbui1

Would it make sense to expand this example and embed somewhere in the current docs, in addition to the storybook documentation?

**Comment by ljharb** [comment]
Yes, that would be excellent.
